sdk/README.md

```diff
@@ -41,7 +47,8 @@ Here we update the `Compiler` of the KFP SDK to generate `Tekton` YAML for a bas
 
 5. Compile the sample pipeline:
 
-    - `cd sdk/samples`  
+    - `mkdir temp && cd temp`
```

Comment:
Linking to external pipeline is okay for complex samples but for basic examples we will need local files in the `sdk/python/tests/compiler/testdata` folder along with the golden output YAML files. I don't think we should get rid of those. So for sequential and parallel join we could point to our local `testdata` folder

Reply:
Hi Christian, I do plan to keep the test files under `sdk/python/tests/compiler/testdata`, but I want to remove `sdk/samples` since it's duplicating with what we have in unittest. For the basic dry run, you think is better to use the unittest files instead of pulling from kfp?

Reply:
I want the instructions on the README to be as short and convenient as possible, trying to avoid any extra `mkdir`, `cd`, `curl` commands.

But I understand the value of not keeping copies of files from other repositories, so maybe we can evolve one good example that is not a copy from another repository. Ideally one that shows some extra value from using Tekton over Argo, but that may be a discussion for another issue :-)